config/services.yml

```diff
@@ -972,9 +969,6 @@ https://cmr.uat.earthdata.nasa.gov:
         bbox: false
         variable: false
       output_formats:
-        - image/tiff
-        - image/png
-        - image/gif
         - application/netcdf # Incorrect mime-type, remove when no longer needed
```

Comment:
This line has intrigued me for a while now (as it has proliferated throughout `services.yml`). I see it was added as part of HARMONY-524. I don't get a sense from that ticket (from Sept 2020) as to when the `application/netcdf` MIME type can be removed entirely in favour of only `application/netcdf-4`.

Reply:
Now I can't remember - was it a QGIS issue? I wish I had left more notes in that ticket. I guess we could remove it and fix anything that breaks.